GDJS/Runtime/pixi-renderers/runtimegame-pixi-renderer.js

```diff
@@ -28,13 +28,21 @@ gdjs.RuntimeGameRenderer = gdjs.RuntimeGamePixiRenderer; //Register the class to
  *
  */
 gdjs.RuntimeGamePixiRenderer.prototype.createStandardCanvas = function(parentElement) {
+    
     //This prevents flickering on some mobile devices
     PIXI.glCore.VertexArrayObject.FORCE_NATIVE = true;
 
-    //Create the renderer and setup the rendering area
+     if (this._game.getScaleMode() === "nearest") {
+        PIXI.settings.SCALE_MODE = PIXI.SCALE_MODES.NEAREST;
+     }else{
+        PIXI.settings.SCALE_MODE = PIXI.SCALE_MODES.LINEAR;
+     }
```

Comment:
Interesting! Somehow missed that when implementing scale mode. Did you observe any difference in the rendering? :)

Reply:
The nearest option currently in gd affect only the parameter image-rendering: pixalate on canvas. 
And affect also all texture loaded. But the parameter is on the resources.
So when i add this on the general rendering i see no difference in preview.

Maybe something can be simplify at this point, Use this setting on general renderer or on each object.
But that is really useful ? (i think, no)

Reply:
These lines are revert in last commit.